src/handlers/RF24DualLogHandler.cpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file RF24DualLogHandler.cpp
+ *
+ * Created on: 30 Mar 2021
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2021        Witold Markowski (wmarkow)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+
+#include <Arduino.h>
+
+#include <handlers/RF24DualLogHandler.h>
+
+RF24DualLogHandler::RF24DualLogHandler(RF24LogHandler *handler1,
+      RF24LogHandler *handler2)
+{
+   this->handler1 = handler1;
+   this->handler2 = handler2;
+}
+
+void RF24DualLogHandler::log(uint8_t logLevel,
+      const __FlashStringHelper *vendorId, const char *message, va_list *args)
+{
+   // va_list can be iterated only once.
+   // Since we have here two handlers, we need to copy arguments.
+   va_list args2;
+   va_copy(args2, *args);
```

Comment:
This was a bit tricky to find, but I have googled it. Basically it works. I have no idea if this has some "hidden side effects".